boot/bootutil/include/bootutil/crypto/aes_ctr.h

```diff
@@ -38,12 +38,47 @@
     #define BOOTUTIL_CRYPTO_AES_CTR_BLOCK_SIZE TC_AES_BLOCK_SIZE
 #endif /* MCUBOOT_USE_TINYCRYPT */
 
+
+#if defined(MCUBOOT_USE_PSA_CRYPTO)
+#include <psa/crypto.h>
+    #include "bootutil/enc_key_public.h"
+    /* To fix: why is it always named BOOTUTIL_CRYPTO_AES_CTR_KEY_SIZE */
+    #define BOOTUTIL_CRYPTO_AES_CTR_KEY_SIZE BOOT_ENC_KEY_SIZE
+    #define BOOTUTIL_CRYPTO_AES_CTR_BLOCK_SIZE (16)
```

Comment:
remove indent

Reply:
Why? The entire file is done this way.

Reply:
We should fix that upstream, it looks silly